src/Paket/Program.fs

```diff
@@ -49,6 +49,9 @@ type CLIArguments =
     | [<AltCommandLine("-f")>] Force
     | Hard
     | [<CustomCommandLine("nuget")>] Nuget of string
+//    | [<CustomCommandLine("gist")>] Gist of string * string
+//    | [<CustomCommandLine("github")>] Github of string * string
+    | [<CustomCommandLine("url")>] Url of string * string
     | [<CustomCommandLine("version")>] Version of string
     | [<CustomCommandLine("add-credentials")>] AddCredentials of string
```

Comment:
I'm gonna need help with this one, guys. As you know, it is possible to reference a single file on Github or an entire repo. Similarly, a single gist file or a multi-file gist. This means that I'd ideally want to use something like `[<CustomCommandLine("gist")>] Gist of string * string option` for parsing `paket add gist`, but I'm getting an exception that Option types are not supported by the UnionArgParser. Any idea how to tackle this problem?